drivers/pinmux/stm32/pinmux_stm32f0.h

```diff
@@ -51,18 +51,42 @@
 #define STM32F0_PINMUX_FUNC_PB11_I2C2_SDA \
 	(STM32_PINMUX_ALT_FUNC_1 | STM32_OPENDRAIN_PULLUP)
 
+#define STM32F0_PINMUX_FUNC_PA4_SPI1_NSS \
+	(STM32_PINMUX_ALT_FUNC_0 | STM32_PUSHPULL_NOPULL)
+#define STM32F0_PINMUX_FUNC_PA5_SPI1_SCK \
+	(STM32_PINMUX_ALT_FUNC_0 | STM32_PUSHPULL_NOPULL)
+#define STM32F0_PINMUX_FUNC_PA6_SPI1_MISO \
+	(STM32_PINMUX_ALT_FUNC_0 | STM32_PUSHPULL_NOPULL)
+#define STM32F0_PINMUX_FUNC_PA7_SPI1_MOSI \
+	(STM32_PINMUX_ALT_FUNC_0 | STM32_PUSHPULL_NOPULL)
+
+#define STM32F0_PINMUX_FUNC_PA15_SPI1_NSS \
+	(STM32_PINMUX_ALT_FUNC_0 | STM32_PUSHPULL_NOPULL)
 #define STM32F0_PINMUX_FUNC_PB3_SPI1_SCK \
 	(STM32_PINMUX_ALT_FUNC_0 | STM32_PUSHPULL_NOPULL)
 #define STM32F0_PINMUX_FUNC_PB4_SPI1_MISO \
 	(STM32_PINMUX_ALT_FUNC_0 | STM32_PUSHPULL_NOPULL)
 #define STM32F0_PINMUX_FUNC_PB5_SPI1_MOSI \
 	(STM32_PINMUX_ALT_FUNC_0 | STM32_PUSHPULL_NOPULL)
 
+/* Available on STM32F030x8 and STM32F030xC devices.*/
```

Comment:
Haven't looked into it, but are they present on other f0 SoCs as well? f091, f072?

Reply:
Maybe, I only looked at f030x specs. We can add support for f09x, f07x later